src/Mappers/InvoiceMapper.php

```diff
@@ -104,7 +104,7 @@ public static function map(Response $response)
             foreach ($lineTags as $tag => $method) {
                 $_tag = $lineDOM->getElementsByTagName($tag)->item(0);
 
-                if (isset($_tag) && isset($_tag->textContent)) {
+                if (isset($_tag) && !empty($_tag->textContent)) {
```

Comment:
I had to change this to prevent errors when setting the performance type because it expects an ENUM but receives an empty string.